### DISCUSSION TITLE
What should I do by step: Do you want to force routing IPv6 to block the leakage?

### DISCUSSION THREAD

**Discussion by ghost** [pivpn/pivpn#1864] [Q&A]
Hello, 
i am installing the lite 64-bit version of bookworm on a RPI 4 with 8 GB.
I have a static IPv4 adress.
So there are no options to enable IPv6 in my router.

Should I go with "yes" or no" at the point "Do you want to force routing IPv6 to block the leakage"
Best regards

**Comment by orazioedoardo** [comment]
Go for yes.
